specs/core/0_beacon-chain.md

```diff
@@ -553,12 +554,12 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 }
 ```
 
-#### `CandidatePoWReceiptRootRecord`
+#### `DepositRootVote`
 
 ```python
 {
-    # Candidate PoW receipt root
-    'candidate_pow_receipt_root': 'hash32',
+    # Deposit root
+    'deposit_root_vote': 'hash32',
```

Comment:
I would just name this `deposit_root`. It is the root that defines this `DepositRootVote`

Reply:
Fixed.